sql/src/main/java/org/apache/druid/sql/calcite/aggregation/builtin/EarliestLatestAnySqlAggregator.java

```diff
@@ -158,7 +162,8 @@ AggregatorFactory createAggregatorFactory(
           case DOUBLE:
             return new DoubleAnyAggregatorFactory(name, fieldName);
           case STRING:
-            return new StringAnyAggregatorFactory(name, fieldName, maxStringBytes);
+          case COMPLEX:
+            return new StringAnyAggregatorFactory(name, fieldName, maxStringBytes, aggregateMultipleValues);
```

Comment:
why would complex map to string? I think this means any type of complex would use the string aggregator which doesn't seem correct to me